src/entity/src/user.js

```diff
@@ -889,13 +889,25 @@ export default class User {
   }
 
   /**
+   * Find users.
+   *
+   * @param {Query} [query] Query used to filter entities.
+   * @param {Object} [options] Options
+   * @param {boolean} [options.discover] Discover users.
+   * @return {Observable} Observable.
+   */
+  static find(query, options = {}) {
```

Comment:
That makes sense, as long as we do it consistently for all `UserStore` calls, and also deprecate the currently documented ones until devs switch over.